### PR TITLE
Fix-ups for dmd 2.071

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Ah, I ought to enable building with dmd 2.071 in this branch as well. Will update.

**Comment by gavin-norman-sociomantic** [comment]
Updated: added commit to build with dmd 2.071.

Green.

**Comment by mihails-strasuns-sociomantic** [comment]
This is extra changes after rebase on latest v14, right?

**Comment by gavin-norman-sociomantic** [comment]
No, it's a prelude to merging v14.x.x -> neo. (See #123.)

**Comment by mihails-strasuns-sociomantic** [comment]
What, why merges again?

**Comment by gavin-norman-sociomantic** [comment]
I've always used merges with neo.

**Comment by nemanja-boric-sociomantic** [comment]
You can update this PR to use swarm `v5.0.11` now?

**Comment by gavin-norman-sociomantic** [comment]
>You can update this PR to use swarm v5.0.11 now?

Done.

File: .travis.yml [hunk 1]
```diff
@@ -23,9 +23,8 @@ env:
         - DMD=1.081.* DIST=xenial F=devel
         - DMD=2.070.2.s* DIST=xenial F=production
         - DMD=2.070.2.s* DIST=xenial F=devel
-        # Depends on turtle being fixed first
-        #- DMD=2.071.2.s* DIST=xenial F=production
-        #- DMD=2.071.2.s* DIST=xenial F=devel
+        - DMD=2.071.2.s* DIST=xenial F=production
+        - DMD=2.071.2.s* DIST=xenial F=devel
```
Comment (nemanja-boric-sociomantic) [inline .travis.yml:27]: As a bonus, you also can enable v2.078.* here. I've just tested it and it works.

Comment (gavin-norman-sociomantic) [reply .travis.yml:27]: Just trying to remember if there was some reason why I didn't do that yet. I think the reason was that this PR is a preparation for merging v14.x.x -> neo. v14.x.x itself doesn't compile with dmd 2.078 yet, so it'd be premature to add support here. I think that was the reasoning. (The merge will add dmd 2.077 support to neo.)

Comment (nemanja-boric-sociomantic) [reply .travis.yml:27]: OK, that sounds fair enough.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**